### PR TITLE
Add RockBand

### PR DESCRIPTION
**Pull request by robosoul**
Added `RockBand` class with `name` method supported.

@codingricky - Changed `rock_band.name` to `rock_band.names` in `en.yml` file to conform to naming conventions (plural for lists). Will revert if it's a problem.

-Luka

### CODE REVIEW

**Comment by codingricky** [comment]
@robosoul It's best to leave the YML file as is because what we do is occasionally take the one from the ruby faker. So if we changed that, we'd had to merge it in as opposed to just replace it. And for a naming/consistency change, it's not really worth it to be honest.

**Comment by robosoul** [comment]
@codingricky - Got it. Reverted.

File: src/test/java/com/github/javafaker/RockBandTest.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.junit.Test;
+
+import static com.github.javafaker.matchers.MatchesRegularExpression.matchesRegularExpression;
+import static org.junit.Assert.assertThat;
+
+public class RockBandTest extends AbstractFakerTest {
+
+    @Test
+    public void name() throws Exception {
+        assertThat(faker.pokemon().name(), matchesRegularExpression("([\\w'\\./ ]+ ?)+"));
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/RockBandTest.java]: That's testing the pokemon service, not the rockband one.

**Comment by robosoul** [comment]
@codingricky - Sorry about that. Fixed now.

**Comment by codingricky** [comment]
@robosoul don't be sorry, I'm very thankful for the help!

**Pull request merged**